muselsl/muse.py

```diff
@@ -285,34 +300,33 @@ def _handle_control(self, handle, packet):
         # Add to current message
         self._current_msg += incoming_message
 
-        if incoming_message[-1] == '}': # Message ended completely
+        if incoming_message[-1] == '}':  # Message ended completely
             self.callback_control(self._current_msg)
 
             self._init_control()
 
     def _subscribe_telemetry(self):
         self.device.subscribe('273e000b-4c4d-454d-96be-f03bac821358',
-                            callback=self._handle_telemetry)
+                              callback=self._handle_telemetry)
 
     def _handle_telemetry(self, handle, packet):
         """Handle the telemetry (battery, temperature and stuff) incoming data"""
 
-        if handle != 26: # handle 0x1a
+        if handle != 26:  # handle 0x1a
             return
         timestamp = self.time_func()
 
         bit_decoder = bitstring.Bits(bytes=packet)
-        pattern = "uint:16,uint:16,uint:16,uint:16,uint:16" # The rest is 0 padding
+        pattern = "uint:16,uint:16,uint:16,uint:16,uint:16"  # The rest is 0 padding
         data = bit_decoder.unpack(pattern)
 
-        packet_index = data[0]
         battery = data[1] / 512
```

Comment:
512 corresponds to what?

Reply:
I honestly can't remember. I would say bit encoding, but 9 bits is weird.